### PR TITLE
1.1/511 improve payment request copy

### PR DESCRIPTION
**Pull request by gingerbreadpup**

### CODE REVIEW

File: database/factories/ModelFactory.php [hunk 1]
```diff
@@ -55,7 +55,7 @@
         'name' => $faker->company,
         'location' => $faker->postcode,
         'sponsor_id' => $sponsor_id, // a random sponsor
-        'payment_message' => "Placeholder Payment Message: $payment_sentence",
+        'payment_message' => "Thanks, your payment request has been accepted. $payment_sentence",
```
Comment (katjam) [inline database/factories/ModelFactory.php]: This is a good candidate for the config/api.php message file. So we don't change in so many places.

Comment (katjam) [inline database/factories/ModelFactory.php]: Could even append to the accessor method for $market->payment_message

**Comment by ob6160** [comment]
Cool - nice accessor method :shipit:

**Comment by ob6160** [comment]
Does this still need the test/comment?

**Pull request merged**